Assets/Plugins/Linux/Core/LinuxPlatformSpecifics.cs

```diff
@@ -20,27 +20,16 @@
 * SOFTWARE.
 */
 
-#if UNITY_64 || UNITY_EDITOR_64
-#define PLATFORM_64BITS
-#elif UNITY_EDITOR_LINUX || UNITY_STANDALONE_LINUX
-#define PLATFORM_32BITS
-#endif
-
-#if UNITY_EDITOR
-#define EOS_DYNAMIC_BINDINGS
-#endif
```

Comment:
Because nearly identical changes to the scripting define conditionals were made in each file where they were altered, I will explain some of the detail here in the first file GitHub indicates is changed by whatever metric they use to determine such things.

- `#define` statements in a Unity C# project are scoped to the file.
- `UNITY_64` is implied by `UNITY_EDITOR_64`, so it is only necessary to check the former.
- `EOS_DYNAMIC_BINDINGS` is in effect an _alias_ for `UNITY_EDITOR`

For all files that made use of `PLATFORM_64BITS` within the project, they were replaced with the `UNITY_64` scripting define.